change/@ni-nimble-angular-d3bfc04f-a6c1-43ac-9d63-8424dd3c1343.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "major",
+  "comment": "Remove enum-text table column",
```

Comment:
Could you update the change files to mention what clients should switch to, just in case someone was using it?